package.json

```diff
@@ -58,6 +58,9 @@
 		"nest:start:debug": "nest start server --debug --watch",
 		"nest:start:prod": "node dist/apps/server/apps/server.app",
 		"nest:start:management": "nest start management",
+		"nest:start:admin-api-server": "nest start admin-api-server",
+		"nest:start:admin-api-server:dev": "nest start admin-api-server --debug --watch",
```

Comment:
For all other places the --debug flag is added by debug. Can you please change the command to 
"nest:start:admin-api-server:debug"
For my personal point of view, it exists no need to add a command with dev that only include --watch without --debug like we have it on other places.